packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -15,7 +15,10 @@ class BuildApkCommand extends BuildSubCommand {
     usesPubOption();
 
     argParser
-      ..addFlag('preview-dart-2', negatable: false,  hide: !verboseHelp)
+      ..addFlag('preview-dart-2',
+          defaultsTo: true,
+          negatable: true,
+	  hide: !verboseHelp)
```

Comment:
indentation style here doesn't match build_aot.dart's style; we should be consistent. :-)